CHANGES.rst

```diff
@@ -498,39 +524,139 @@ Other changes
+.. _2306: https://github.com/cython/cython/issues/2306
+.. _3468: https://github.com/cython/cython/issues/3468
+.. _3332: https://github.com/cython/cython/issues/3332
+.. _3202: https://github.com/cython/cython/issues/3202
+.. _3188: https://github.com/cython/cython/issues/3188
+.. _3179: https://github.com/cython/cython/issues/3179
+.. _2891: https://github.com/cython/cython/issues/2891
+.. _2826: https://github.com/cython/cython/issues/2826
+.. _2713: https://github.com/cython/cython/issues/2713
+.. _3365: https://github.com/cython/cython/issues/3365
+.. _2748: https://github.com/cython/cython/issues/2748
+.. _2580: https://github.com/cython/cython/issues/2580
+.. _3413: https://github.com/cython/cython/issues/3413
+.. _2855: https://github.com/cython/cython/issues/2855
+.. _2701: https://github.com/cython/cython/issues/2701
+.. _2889: https://github.com/cython/cython/issues/2889
+.. _3489: https://github.com/cython/cython/issues/3489
+.. _3307: https://github.com/cython/cython/issues/3307
+.. _3391: https://github.com/cython/cython/issues/3142
+.. _2963: https://github.com/cython/cython/issues/2963
+.. _1807: https://github.com/cython/cython/issues/1807
+.. _3100: https://github.com/cython/cython/issues/3100
+.. _2941: https://github.com/cython/cython/issues/2941
+.. _2972: https://github.com/cython/cython/issues/2972
+.. _2752: https://github.com/cython/cython/issues/2752
+.. _2536: https://github.com/cython/cython/issues/2536
+.. _2133: https://github.com/cython/cython/issues/2133
+.. _3244: https://github.com/cython/cython/issues/3244
+.. _1635: https://github.com/cython/cython/issues/1635
+.. _2791: https://github.com/cython/cython/issues/2791
+.. _2665: https://github.com/cython/cython/issues/2665
+.. _2968: https://github.com/cython/cython/issues/2968
+.. _2775: https://github.com/cython/cython/issues/2775
+.. _3106: https://github.com/cython/cython/issues/3106
+.. _3102: https://github.com/cython/cython/issues/3102
+.. _3406: https://github.com/cython/cython/issues/3406
+.. _2565: https://github.com/cython/cython/issues/2565
+.. _2772: https://github.com/cython/cython/issues/2772
+.. _2790: https://github.com/cython/cython/issues/2790
+.. _2904: https://github.com/cython/cython/issues/2904
+.. _2905: https://github.com/cython/cython/issues/2905
+.. _2952: https://github.com/cython/cython/issues/2952
+.. _3001: https://github.com/cython/cython/issues/3001
+.. _2686: https://github.com/cython/cython/issues/2686
+.. _3355: https://github.com/cython/cython/issues/3355
+.. _`PEP-560`: https://www.python.org/dev/peps/pep-0560
+.. _`PEP-570`: https://www.python.org/dev/peps/pep-0570
+.. _`PEP-487`: https://www.python.org/dev/peps/pep-0487
+.. _`PEP-590`: https://www.python.org/dev/peps/pep-0590
+.. _`PEP-3131`: https://www.python.org/dev/peps/pep-3131
+.. _`PEP-563`: https://www.python.org/dev/peps/pep-0563
+.. _`PEP-479`: https://www.python.org/dev/peps/pep-0479
+
+.. _0.29.22:
 
 0.29.22 (2021-02-20)
```

Comment:
If any, then all versions should have an anchor like this. Again, might be something that Sphinx could do.

Reply:
The link without the anchor is `0.29.22-(2021-02-20)`, I could change the reference to that text instead.

Reply:
I dropped the anchors and the references. It is not worth the effort to mess with this: sphinx does not do a good job creating anchors with only digits, periods, spaces, and parentheses.